purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -41,52 +43,60 @@ data class PaywallData(
     /**
      * Returns the [LocalizedConfiguration] to be used based on the current locale.
      */
-    val localizedConfiguration: LocalizedConfiguration
+    val localizedConfiguration: Pair<Locale, LocalizedConfiguration>
```

Comment:
I changed this to also return the locale selected. This is a bit weird as a public API, but I think we should use this instead of the default locale in case the default locale doesn't match the one we use for the strings. This way we can make sure to use the same locale everywhere.

Reply:
Makes sense. I'd update the docstring above though.